src/ctl/src/common/context.rs

```diff
@@ -63,10 +64,13 @@ impl CtlContext {
     pub async fn try_close(mut self) {
         tracing::info!("clean up context");
         if let Some(meta_client) = self.meta_client.take() {
-            if let Err(e) = meta_client
-                .unregister(meta_client.host_addr().clone())
-                .await
-            {
+            let addr = meta_client.host_addr().clone();
```

Comment:
The meta client probably doesn't need to cordon itself when exiting, right?

Reply:
Yes. You are right.